buzzwindow.cpp

```diff
@@ -5,33 +5,45 @@
 #include "enum.h"
 #include <Application.h>
 #include "main.h"
+#include <GroupLayout.h>
+#include <GroupLayoutBuilder.h>
+#include <LayoutBuilder.h>
+
 //-------------------------------------------------------------------
 BuzzWindow::BuzzWindow()
-	: BWindow(BRect(30, 30, 150, 80), "WakeUp v1.0", B_TITLED_WINDOW, B_NOT_RESIZABLE | B_NOT_ZOOMABLE)
+	: BWindow(BRect(30, 30, 0, 0), "WakeUp v1.0",
```

Comment:
The `:` should be on its own line. Please refer to [Haiku Coding Guidelines](https://www.haiku-os.org/development/coding-guidelines/).